lib/http.framework.express.js

```diff
@@ -307,6 +307,11 @@ HttpFrameworkExpress.prototype.addWrappedMethodFunction = function (method, midd
 
     handler(this.buildInputs(req), req.session, req.cookies, req, res, next)
       .then(function (output) {
+        // do nothing if a response has already been sent
+        if (res.headersSent) {
+          return;
+        }
```

Comment:
Why not just skip the writing of the headers?
This will stop body writes as well.

Reply:
Thanks for catching this, I'll make the change.